packages/transformer/.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "args": [
+                "--timeout",
+                "800000",
+                "--require",
+                "source-map-support/register",
+                "${workspaceFolder}/lib/cjs/test/**/*.test.js"
+            ],
+            "internalConsoleOptions": "openOnSessionStart",
+            "name": "Mocha Tests",
+            "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
```

Comment:
maybe it's worth just having this run `npm run test` so we don't need to keep the two debugging script in sync if we change mocha configuration?